src/Launcher/Launcher.jl

```diff
@@ -641,8 +647,8 @@ function launch!(
         # convert J for type safety
         J = Vector{Vector{Float64}}([[x...] for x in J])
 
-        # normalize couplings
-        normalize!(J)
+        # normalize couplings and level repulsion
+        J, A = normalize([J, [[A]]])
```

Comment:
Does this work? If we do 

```
J      = [rand(2), rand(2)]
A     = rand()
J, A = normalize([J, [[A]]])
```

Then `typeof(A)` returns `Vector{Vector{Float64}}`, which is inconsistent with `set_repulsion.`

Reply:
This is not the final version... If you look at the final diffs, there types match